src/Compilers/CSharp/Test/Emit3/Semantics/CollectionExpressionTests_WithElement_Extra.cs

```diff
@@ -4026,6 +5323,314 @@ static T Identity<T>(T value)
             Diagnostic(ErrorCode.ERR_CollectionBuilderAttributeMethodNotFound, "[with(y: Identity(1), x: Identity(2)), Identity(3), Identity(4)]").WithArguments("Create", "T", "MyCollection<T>").WithLocation(7, 13));
     }
 
+    [Fact]
+    public void EvaluationOrder_CollectionBuilder_A()
```

Comment:
In test above, why keep the link to Chuck's issue? It doesn't seem applicable since different implementation. Please scan through all the links for tests you enable in this PR to check if they are applicable. We can close the dead ones